### PR TITLE
1299643: Update virt-who-config man page to include NO_PROXY

### PR DESCRIPTION
**Pull request by cnsnyder**

### CODE REVIEW

File: virt-who-config.5 [hunk 1]
```diff
@@ -2,10 +2,12 @@
 .SH NAME
 virt-who-config - configuration for virt-who
 .SH SYNOPISIS
+/etc/sysconfig/virt-who
 /etc/virt-who.conf
 /etc/virt-who.d/*
 .SH DESCRIPTION
-Configuration format is ini-like.
+Configuration format is ini-like for files /etc/virt-who.conf and /etc/virt-who.d/*.
+The contents of /etc/sysconfig/virt-who are environment variables to be used when virt-who is run as a service.
```
Comment (barnabycourt) [inline virt-who-config.5]: How id running virt-who as a service (/etc/sysconfig/virt-who) different than normal (
/etc/virt-who.d/*)?

File: virt-who-config.5 [hunk 2]
```diff
@@ -43,6 +45,10 @@ The file name to write logs to (used only if log_per_config=False)
 A list of files containing configurations for virt-who
 Used to specify locations other than default
 
+.SH VARIABLES UNIQUE TO SYSCONFIG
+.TP
+\fBNO_PROXY\fR
+A comma-separated list of hostnames or domains or ip addresses to ignore proxy settings for.
```
Comment (barnabycourt) [inline virt-who-config.5]: Needs to include information about using wildcards

File: virt-who-config.5 (hunk 1 above)
Comment (cnsnyder) [inline virt-who-config.5]: When run as a service the variables in /etc/sysconfig/virt-who are added to the environment of the virt-who process. This is yet another means of configuring virt-who.

File: virt-who-config.5 (hunk 2 above)
Comment (cnsnyder) [inline virt-who-config.5]: Updated.

**Comment by barnabycourt** [comment]
@cnsnyder ACK, this looks good. I don't have write access so I can't merge the PR for you.

**Pull request merged**